docs/basic_usage.md

```diff
@@ -0,0 +1,62 @@
+# DAPper Basic Usage Guide
+
+DAPper (Dependency Analysis Project) is a tool that analyzes C/C++ source code to discover implicit dependencies and identify which system packages are required. This guide covers the basic usage patterns and common workflows.
+
+## Overview
+
+DAPper helps you:
+- **Discover header dependencies**: Find all `#include` statements in your C/C++ code
+- **Identify system packages**: Map header files to the packages that provide them
```

Comment:
This could maybe be removed if it's rolled into the previous bullet points description -- or we could be more of a description like "the datasets can be used to figure out what packages were most likely to have provided a given file".